.github/workflows/ui-e2e.yaml

```
@@ -0,0 +1,520 @@
+        run: |
+          kill "$MONITOR_PID" 2>/dev/null || true
+          kind delete cluster --name "$CLUSTER_NAME" 2>/dev/null || true
+
+  # ── GKE mode: shared cluster, scanner v4, all specs ──────────────────
+  provision-gke:
+    name: Provision GKE
+    if: contains(github.event.pull_request.labels.*.name, 'ui-e2e-gke')
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    outputs:
+      cluster-name: ${{ steps.cluster.outputs.cluster-name }}
```

actionlint (1.7.12)

[error] property "cluster-name" is not defined in object type {}

(expression)